src/modules/esl-popup/core/esl-popup.ts

```diff
@@ -174,6 +170,21 @@ export class ESLPopup extends ESLToggleable {
     memoize.clear(this, '_offsetArrowRatio');
   }
 
+  /**
+   * Actions to execute after show popup.
+   */
+  protected afterOnShow(): void {
+    this._updatePosition();
+    this.style.visibility = 'visible';
+    this.activator && this._addActivatorObserver(this.activator);
+    this._startUpdateLoop();
+  }
+
+  /**
+   * Actions to execute before hide popup.
```

Comment:
Comment sounds more correct like this: "Actions to execute after popup hide" or "Actions to execute after hide of popup"